spec/models/order_spec.rb

```diff
@@ -7,6 +7,12 @@
     it { is_expected.to be_valid }
   end
 
+  describe "default order status" do
+    it "should a default status fo pending" do
```

Comment:
**Nitpick - Code Quality**
`"should have a default status of pending"`. Please correct the grammar above.